ui/src/views/infra/UsageRecords.vue

```diff
@@ -50,6 +50,7 @@
           <a-row justify="end">
             <a-col>
               <tooltip-button
+                v-if="'generateUsageRecords' in this.$store.getters.apis"
```

Comment:
minor nit: there's no need to use `this` here
```suggestion
                v-if="'generateUsageRecords' in $store.getters.apis"
```